Brewfile

```diff
@@ -9,10 +9,25 @@ tap 'homebrew/services'
 
 brew 'hub'
 
+brew 'awscli'
+brew 'gpg'
+brew 'opensc'
+brew 'jq'
+brew 'packer'
+brew 'docker'
+brew 'docker-machine'
+brew 'ykman'
+brew 'coreutils'
+brew 'rbenv'
+
 tap 'cloudfoundry/homebrew-tap'
 brew 'cf-cli'
 
 cask 'chromedriver'
 cask 'docker'
 cask 'slack'
 
+cask 'chef/chef/chefdk'
```

Comment:
this and some of the other items are rather login.gov specific. I guess that's not an issue given that we seem to be the only team using this still but something to keep in mind in case this get's used by a wider audience.